src/components/Button/useButtonStyles.tsx

```diff
@@ -14,6 +14,8 @@ export const getThemeColor = (color: ButtonColorVariant): ThemeColor => {
       return 'violet-400';
```

Comment:
should we opt to allow all colors (i.e. `teal-500` instead of `teal`) or have you synced with @dimitriszoitas and he said that we won't be adding more color variants in the near future?

Reply:
He is definitely going to add more colors in the future. I can delete the theme colors from the Button and allow all colors, we'll just have to update all panther buttons when this gets merged.

Reply:
yeah, let's not do that now, but should track it in a ticket. Can you create one @kouknick ?

Reply:
Done!